Description: Remove Go build dependencies for pre-built binaries
 Since we use pre-built RISC-V64 binaries from upstream releases, we don't
 need any Go toolchain or library dependencies. This dramatically simplifies
 the build-depends list.
Author: Bruno Verachten <gounthar@gmail.com>
Forwarded: not-needed
Last-Update: 2025-10-18

--- a/debian/control
+++ b/debian/control
@@ -5,100 +5,6 @@
 Maintainer: Bruno Verachten <gounthar@gmail.com>
 Build-Depends: debhelper-compat (= 13)
               ,dh-apparmor
               ,bash-completion
               ,ca-certificates
-              ,dh-golang (>= 1.14~)
-              ,golang-any (>= 2:1.10~)
-              ,go-md2man (>= 1.0.3~)
-              ,golang-dbus-dev (>= 5.0.2~)
-              ,golang-github-containerd-containerd-dev (>= 1.7.23~)
-              ... (removing ~100 golang-* dependencies)
 Homepage: https://github.com/moby/moby
 Vcs-Git: https://github.com/gounthar/docker-for-riscv64.git
